docs/advanced-features/clearing-the-incremental-cache.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Next.js has the abilility to clear the incremental cache by array of keys. You can learn how it works here.
```

Comment:
```suggestion
description: Next.js has the ability to clear the incremental cache by array of keys. You can learn how it works here.
```

Reply:
This was a copy/paste job from another part of the docs. I'll find that other section as well and PR the fix.